### DISCUSSION TITLE
Scrum Prep Milestone 2-3

### DISCUSSION THREAD

**Discussion by NatiPerez459** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#138] [General]
Each team member will comment progress made through the past week.

**Comment by DiegoVazquez1** [comment]
Está semana cree y termine el task de hacer el cart page ui. Para eso cree el vue file que incluye todos los feature del total cost y products. Todos los botones funcionan excepto el checkout ya que será otra página aparte

**Comment by CovasJJ** [comment]
This week I created and finished the task of revising sections 2.1.2 and 2.1.3 of the documentation. I was able to use the professor's feedback as a guide to complete my task.

**Comment by JadStelar** [comment]
I have been working on refining the task for the implementation of Pass and familiarizing myself more on its implementation.
-Jahdiel Montero

**Comment by JuanDelPueblo** [comment]
This week I connected the MongoDB database to the login and signup pages, ensuring that authentication works without storing any data locally.

**Comment by RyanVelez101** [comment]
This week I created and finished the task for the functionality of the home, products, and search buttons. 
-Ryan Velez

**Comment by Kaariinaa08** [comment]
Developed a responsive product listing search page following a previously provided Sketch design. Implemented key features such as a search bar, collapsible filter sidebar, pagination, product sorting options, and both grid and list layouts. Ensured all components adhered to design specifications for functionality, visual styling, and responsiveness.
- Karina Lopez Rodriguez

**Comment by cristian-barreras** [comment]
The task of reCAPTCHA was tested and completed.

**Comment by Christian2101** [comment]
This week started fixing parts of the documentation in relation to the professors feedback.

**Comment by Cabinetman** [comment]
This week, I focused on implementing the Order event entity. I designed the entity structure and integrated it according to the initial ERD.

**Comment by elliot-gomez** [comment]
This week, I created a document compiling all of the professor’s feedback on the Milestone 1 documentation. I organized the feedback by the specific sections it applied to, making it clear for the team which feedback needs to be addressed in each part of the document. In addition to organizing the feedback, I also created tasks for revising the documentation based on this feedback. I have taken on several of these tasks and have been actively working on making the necessary revisions to the documentation.

- Elliot J. Gomez Coriano

**Comment by AlvinBadillo** [comment]
This week, I completed the Order controller with full CRUD functionality. It manages user associations and order totals, with validation and error handling based on the Item controller. The controller is now operational and meets all requirements.

**Comment by PerezMarcelo** [comment]
This week I created a CAPTCHA from scratch and implemented it into the client and seller logins. Also, I monitored and overviewed developers tasks.

**Comment by F3rnand0hr** [comment]
This week, I edited the favicon and title of the website. Started investigating on other features we can implement in the future.

**Comment by gcal77** [comment]
This week I worked on providing a draft for a better UI design for the client login page and i'm also working on implementing lazy loading image functionality on the home page.

**Comment by franrov4** [comment]
Busy week, couldn't get much work done with regards to the project. Kept contact with team leads, looking to create some tasks to push forward the progress towards the backend-db connection.
